src/include/platform/internal/GenericPlatformManagerImpl.cpp

```diff
@@ -102,6 +102,16 @@ CHIP_ERROR GenericPlatformManagerImpl<ImplClass>::_InitChipStack()
     }
     SuccessOrExit(err);
 
+    // Initialize the NFC Manager.
+#if CHIP_DEVICE_CONFIG_ENABLE_NFC
+    err = NFCMgr().Init();
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(DeviceLayer, "NFC Manager initialization failed: %s", ErrorStr(err));
```

Comment:
How about `VerifyOrExit(err, ChipLogError(DeviceLayer, "NFC Manager initialization failed: %s", ErrorStr(err)))` ?

Reply:
Yeah, I wanted to be consistent with the existing code, but can change it.